docs/code/StatisticsBackendTests.cpp

```diff
@@ -227,6 +227,32 @@ void get_data_examples()
     }
 }
 
+void get_data_supported_entity_kinds_examples()
+{
+    {
+        EntityId host1_id;
+        EntityId host2_id;
+
+        //CONF-GET-FASTDDS-LATENCY-SUPPORTED-ENTITY-KINDS
+        /* Get all the EntityKind pairs related to DISCOVERED_ENTITY. */
+        auto types_list = StatisticsBackend::get_data_supported_entity_kinds(DataKind::DISCOVERY_TIME);
+
+        /* Iterate over all the valid pairs composing the final result */
+        std::vector<StatisticsData> discovery_times;
+        for (auto types : types_list)
```

Comment:
`types` might be a reserved word. You could call it `type_pair` or something like that